cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -803,14 +782,14 @@ public void testNewKeyGetGeneratedWhenInRenewalPeriodAuto(VertxTestContext conte
                 assertThat(newClusterCaKey, is(notNullValue()));
                 assertThat(newClusterCaKey, is(not(initialClusterCaKeySecret.getData().get(CA_KEY))));
 
-                Map<String, String> clientsCaCertData = c.getAllValues().get(2).getData();
+                Map<String, String> clientsCaCertData = secretData(c, 2);
                 assertThat(clientsCaCertData, aMapWithSize(4));
                 X509Certificate newX509ClientsCaCertStore = getCertificateFromTrustStore(CA_CRT, clientsCaCertData);
                 String oldClientsCaCertKey = clientsCaCertData.keySet()
                         .stream()
                         .filter(alias -> alias.startsWith("ca-"))
                         .findAny()
-                        .get();
+                        .orElseThrow();
```

Comment:
`Optional#orElseThrow` instead of `Optional#get` removes the warning about calling `Optional#get` without calling `Optional#isPresent` first.

Reply:
:+1:

Reply:
Kia ora @tombentley, FYI, I've created a draft PR around Optionals that I'll ready for review once this PR is merged. 
https://github.com/strimzi/strimzi-kafka-operator/pull/5243